.github/workflows/wildfly-build.yml

```diff
@@ -37,7 +43,7 @@ jobs:
           cache: 'maven'
       - name: Build with Maven Java ${{ matrix.java }} on WildFly ${{needs.wildfly-build.outputs.wildfly-version}}
         run: |
-          mvn -s ./.m2-settings.xml -B -V -fae -Pwildflydev clean install
+          mvn -s ./.m2-settings.xml -B -V -fae -Pwildflydev clean install '-Dwildfly.version=${{needs.wildfly-build.outputs.wildfly-version}}'
```

Comment:
missed one `wildflydev`

Reply:
There is still one profile left which I assume is needed https://github.com/jbossws/jbossws-cxf/blob/004e5a30e87b7c991f2e11fd087bc7bcfca9c660/modules/testsuite/pom.xml#L671-L755.

Reply:
Never mind, I see what you said above now :)